.github/actions/composite/setupNode/action.yml

```diff
@@ -27,19 +33,26 @@ runs:
         path: node_modules
         key: ${{ runner.os }}-node-modules-${{ hashFiles('package-lock.json', 'patches/**') }}
 
+    - id: cache-old-dot-node-modules
+      if: inputs.IS_HYBRID_BUILD == 'true'
+      uses: actions/cache@v4
+      with:
+        path: Mobile-Expensify/node_modules
+        key: ${{ runner.os }}-node-modules-${{ hashFiles('Mobile-Expensify/package-lock.json', 'Mobile-Expensify/patches/**') }}
+
     - id: cache-desktop-node-modules
       uses: actions/cache@v4
       with:
         path: desktop/node_modules
         key: ${{ runner.os }}-desktop-node-modules-${{ hashFiles('desktop/package-lock.json', 'desktop/patches/**') }}
 
     - name: Install root project node packages
-      if: steps.cache-node-modules.outputs.cache-hit != 'true'
+      if:  (steps.cache-node-modules.outputs.cache-hit != 'true') || (inputs.IS_HYBRID_BUILD == 'true' && steps.cache-old-dot-node-modules.outputs.cache-hit != 'true') 
```

Comment:
Hey @jnowakow, just to check:
-  if there is no cache then we wipe node modules? Isn't that redundant or do I misunderstand (condition 1)
- OR if this is a HybridApp build and there is no OldDot cache we also wipe node modules (makes sense)
- but do we also need to wipe HybridApp cache somewhere?

Reply:
It looks like we are wiping NewDot node_modules to me, but I must be mistaken?